routes/api/v1/dashboards.js

```diff
@@ -0,0 +1,46 @@
+const Dashboard = require( '../../../models/dashboards.js' )
+const { Widget } = require( '../../../models/widgets.js' )
+const addWidget = require( './dashboard/addWidget.js' )
+const errorResponse = require( './errorResponse.js')
+
+const express = require( 'express' )
+const router = express.Router()
+
+router.get( '/', ( request, response ) => {
+  Dashboard.find().exec()
+    .then( dashboard => response.json( dashboard ) )
+    .catch( errorResponse( response ))
+})
+
+router.post( '/', ( request, response ) => {
+  Dashboard.create( request.body )
+    .then( dashboard => response.json( dashboard ))
```

Comment:
Successful creation (and a few of the other operations) should use different HTTP status codes - `201 Created`, for example, when a new item is created, and `204 No Content` when an item is deleted.